backend/src/models/db/courses.js

```diff
@@ -0,0 +1,13 @@
+const data = require('../../../../frontend/public/course-names.json')
```

Comment:
Just confirming this is a temporary parser and we will eventually will have all this data in a permanent db?